drivers/sub8_videoray_m5_thruster/readme.md

```diff
@@ -11,10 +11,11 @@ It is now maintained by David Soto.
 This node handles the interface between the sub's control computer and the thrusters. It communicates via serial to each thruster, individually issuing commands, it also receives response packets from each of the thrusters (including current, voltage, rpm, faults, etc.) and publishes them.
 
 # Calibration
-The thrusters take an input on [-1, 1]. The calibration.json file provided maps commanded effort values to produced output thrust in Newtons.
+The thrusters take an input on [-1, 1]. The files provided  in `config/calibration/` map commanded effort values to expected output thrust in Newtons.
```

Comment:
If the Jason propellers ever make a comeback, can we please name the calibration file `jason.json`?